code/fighter.py

```diff
@@ -107,8 +118,12 @@ def update(self): # TODO: ADD UPDATE FUNCTION
        pass
 
     def ai(self, obstacle_list):
-        # if self.alive and player.alive:
+        """Initialises movement for the enemies
 
+         :param obstacle_list: A list with all obstacles
```

Comment:
Are obstacles objects in the map that the enemies can collide with?

Reply:
Thx for the comment! I indeed described that rather vaguely, my bad: 
:param obstacle_list: A tuple containing a list of all obstacles. Which is tile information for only things that
         are seen as obstacles (e.g., things the player can walk into like a wall, or the ground). This list includes
          an image of the obstacle and the position of the obstacle.

Does this make it more clear?